addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -72,6 +73,54 @@ export class AnalyticDistribution extends Component {
         this.allPlans = [];
         this.lastAccount = this.props.account_field && this.props.record.data[this.props.account_field] || false;
         this.lastProduct = this.props.product_field && this.props.record.data[this.props.product_field] || false;
+
+        this.activeTag = null;
+        this.selectCreateIsOpen = false;
```

Comment:
Maybe instead of storing this on the model, we could check if the element clicked on is a child of a `.modal` css selector element?

Reply:
when the selectCreate dialog is open, the popup needs to stay open (for the reason mentioned above)
so we'd have to use a document.QuerySelector to determine whether the selectCreate dialog is open... far from ideal
keeping a variable to track whether it's open and ignore window clicks is substantially cleaner